### PR TITLE
Millstone [1] : Apple Clone

### PR DESCRIPTION
**Pull request by MohameDawood**
In this project we were required to create a clone of an old apple website (https://web.archive.org/web/20140228175622/http://www.apple.com).

We also did all code organized according to BEM methodology.

### CODE REVIEW

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @MohameDawood and @lucianosarno! 👋 👋 
You are doing a great job with this project! 👍 You are almost done!  🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Your pull request has a great title, however, there is no description. **It is extremely important that you add a short description of what you have done in your pull request.** Please add this before requesting for your next review. 

- [x] There is no live demo link in your README file. Please add one. You can generate this using [this link](https://raw.githack.com/).

- [x] The elements in your navigation bar should be links. Please see my comments below. 
Screen Shot 2020-06-23 at 8 44 37 PM

- [ ] Additionally, the gradient background of your navigation bar appears to be an image. Please implement this using the` linear-gradient` CSS property. You can find more information about this [here](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient) and [here.](https://www.w3schools.com/css/css3_gradients.asp).
Screen Shot 2020-06-23 at 8 48 06 PM


- [x] The four images directly below the main section should be links. Please see my comments below.Screen Shot 2020-06-23 at 8 49 06 PM


****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                <h1 class="photo1__phrase1">What will your verse be?</h1>
+                <nav class="mainMenu photo1__mainMenu">
+                    <button type="button" class="mainMenu__button mainMenu__button_apple"></button>
+                    <button type="button" class="mainMenu__button">Store</button>
+                    <button type="button" class="mainMenu__button">Mac</button>
+                    <button type="button" class="mainMenu__button">iPod</button>
+                    <button type="button" class="mainMenu__button">iPhone</button>
+                    <button type="button" class="mainMenu__button">iPad</button>
+                    <button type="button" class="mainMenu__button">iTunes</button>
+                    <button type="button" class="mainMenu__button">Support</button>
```
Comment (BrittanyBlake) [inline index.html]: The elements in the navigation bar should be links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <h2 class="photo1__phrase1">What will be your verse be?</h2>
+                <h2 class="photo1__phrase2">Each of us has something to share. A voice, a passion, a perspective. The potential to add a stanza to the world’s story. We were so inspired by how people use iPad every day, we set out to capture a few of these moments.</h2>
+                <h3 class="photo1__phrase3">Watch the “Your Verse” film and explore the stories</h3>
+            </section>
+            <section class="smallerPhotos main__smallerPhotos">
+                <h1 class="smallerPhotos__h1">Highlights</h1>
+                <article class="smallerPhotos__smallerPhotosArticle smallerPhotos__smallerPhotosArticle_photo1"></article>
+                <article class="smallerPhotos__smallerPhotosArticle smallerPhotos__smallerPhotosArticle_photo2"></article>
+                <article class="smallerPhotos__smallerPhotosArticle smallerPhotos__smallerPhotosArticle_photo3"></article>
+                <article class="smallerPhotos__smallerPhotosArticle smallerPhotos__smallerPhotosArticle_photo4"></article>
```
Comment (BrittanyBlake) [inline index.html]: The elements below the main section should be links.

File: photo1.css [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+.mainMenu {
+  width: 980px;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  height: 36px;
+  padding: 0;
+  border-radius: 4px;
+  background-image: url(http://images.apple.com/global/nav/images/globalheader.png);
```
Comment (BrittanyBlake) [inline photo1.css]: The background gradient should be implemented using the `linear-gradient` property. Please remove the background image here.

File: README.md [hunk 4]
```diff
@@ -1 +1,47 @@
-# appleClone
\ No newline at end of file
+# Apple Clone
+> Luciano and Mohamed 4th project in HTML&CSS3 module.
+
+![screenshot](./#)
```
Comment (BrittanyBlake) [inline README.md]: Please add a screenshot of your project.

File: README.md [hunk 5]
```diff
@@ -1 +1,47 @@
...
+In this task we were required to create a clone of an Apple old website.
+
+## Built With
+
+- HTML,
+- CSS3,
+
+## Live Demo
+
+[Live Demo Link](#)
```
Comment (BrittanyBlake) [inline README.md]: Please add a live demo

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved :handshake: 

### Hi @MohameDawood  and @lucianosarno 

### You did great with the project, you implemented all the changes required from the previous TSE. :+1: 
### You can merge this branch into master

**Pull request merged**